### PR TITLE
add pekko grpc support

### PR DESCRIPTION
**Pull request by pjfanning**
* add pekko example

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -973,6 +973,23 @@ lazy val grpcExamples: ProjectMatrix = (projectMatrix in file("grpc/examples"))
     akkaGrpcServer
   )
 
+lazy val pekkoGrpcExamples: ProjectMatrix = (projectMatrix in file("grpc/pekko-examples"))
```
Comment (adamw) [inline build.sbt:976]: needs to be added to aggregates :)

Comment (pjfanning) [reply build.sbt:978]: added now

**Comment by pjfanning** [comment]
> Now it seems there are some compile problems (in the scala 3 + jvm version)

I've changed the pekko-rpc example to not support Scala 3 - pdDirectProtobuf is not ready for scala 3

**Comment by adamw** [comment]
@mergifyio rebase

**Comment by adamw** [comment]
Somehow this breaks the JS builds ... maybe the sbt plugin updates some guava components?

**Comment by pjfanning** [comment]
@adamw the guava dependency seems to come from io.grpc:grpc-core:1.48.1. akka-grpc also relies on io.grpc:grpc-core.

So I am not sure why this build issue only seems to affect pekko-grpc.

**Comment by adamw** [comment]
@mergifyio rebase

**Comment by pjfanning** [comment]
Would it make sense to update the build scripting to exclude pekko-grpc from the javascript build?

**Comment by adamw** [comment]
Unfortunately the entier build is failing, not only the grpc modules. The exception is from the `catsJS2_12` module, completely unrelated to grpc

**Comment by kciesielski** [comment]
I added a workaround for the guava issue, also proposed a fix in pekko-grpc. Let's see if it helps :)

**Pull request merged**